app/src/components/misc/SupportPanel/index.tsx

```diff
@@ -0,0 +1,192 @@
+        <a
+          href="https://www.reddit.com/r/requestly/"
+          target="_blank"
+          rel="noreferrer"
+          className="support-panel-item-style"
+          onClick={() => {
+            setVisible(false);
+            trackEvent("support_option_clicked", {
+              type: "reddit",
+            });
+          }}
+        >
+          <span className="support-panel-label-style">Ask us on Reddit</span>
+        </a>
+      ),
+      icon: <RedditOutlined />,
+    },
+    {
+      key: "3",
+      label: (
+        <div
+          className="support-panel-item-style"
+          onClick={() => {
+            setVisible(false);
+            setIsRequestBotVisible(true);
+            trackEvent("support_option_clicked", {
+              type: "ask_ai",
+            });
+          }}
+        >
+          <span className="support-panel-label-style">
+            Ask AI<i style={{ color: "#bbbbbb" }}> 24x7 Support</i>
+          </span>
+        </div>
+      ),
+      icon: <AILogo />,
+    },
+    {
+      key: "4",
+      label: (
+        <div
+          className={`support-panel-item-style ${!paidUser && "support-panel-item-disabled"}`}
+          onClick={() => {
+            if (paidUser) {
+              window.$crisp.push(["do", "chat:open"]);
+              trackEvent("support_option_clicked", {
+                type: "chat_with_us",
+              });
+            } else {
+              dispatch(
+                // @ts-ignore
+                actions.toggleActiveModal({
+                  modalName: "pricingModal",
+                  newValue: true,
+                  newProps: { selectedPlan: null, source: "support_option_clicked" },
+                })
+              );
+              trackEvent("support_option_clicked", {
+                type: "upgrade_to_chat_with_us",
+              });
+            }
+            paidUser
+              ? window.$crisp.push(["do", "chat:open"])
+              : dispatch(
+                  // @ts-ignore
+                  actions.toggleActiveModal({
+                    modalName: "pricingModal",
+                    newValue: true,
+                    newProps: { selectedPlan: null, source: "support_option_clicked" },
+                  })
+                );
+          }}
+        >
+          <span className="support-panel-label-style ">
+            Chat With Us
+            {!paidUser && <span className="support-panel-upgrade-badge-style">Upgrade</span>}
+          </span>
+        </div>
+      ),
+      icon: <MessageIcon />,
+    },
+    !slackConnect &&
+      !slackInviteVisibilityStatus &&
+      user.loggedIn &&
+      !paidUser && {
+        key: "5",
+        label: (
+          <a
+            href="mailto:contact@requestly.io"
+            target="_blank"
+            rel="noreferrer"
+            className="support-panel-item-style"
+            onClick={() => {
+              setVisible(false);
+              trackEvent("support_option_clicked", {
```

Comment:
1. all trackEvent calls should be wrapped into a function 
2. all these related event call functions (ones that are related to the same component or feature) must be moved to one file
3. event names should never be used as literals like this, we should rather create constants and then use the constants.

for eg. refer [this](https://github.com/requestly/requestly/blob/1114649045af60df669926d7e5ab818fa1c6d643/app/src/modules/analytics/events/common/auth/signup/index.js) or [this](https://github.com/requestly/requestly/tree/48c3abf3a50b709d0d2203c300eef0034fdd2d15/app/src/modules/analytics/events/features/sessionRecording) (_or any other event file in the app_)

Reply:
do this for all your trackEvent calls